### PR TITLE
Task 52982 fixes

### PR DESCRIPTION
**Pull request by TirlaP**

### CODE REVIEW

File: src/components/common/card/Card.js [hunk 1]
```diff
@@ -1,44 +1,94 @@
...
+        isFavorite,
+        last,
+        title,
+        location,
+        price,
+        image,
+      })
+    );
+    event.preventDefault();
+    console.log(isFavorite);
```
Comment (BiancaDianaAlexiuc) [inline src/components/common/card/Card.js:43]: You can remove the console.log

**Review by BiancaDianaAlexiuc — APPROVED** [review]

**Review by petricafinaga — APPROVED** [review]

File: src/components/common/card/CardStyle.js [hunk 2]
```diff
@@ -1,51 +1,83 @@
...
+	width: 92,
+	color: "primary",
+	fontSize: "14px",
+	fontWeight: "600",
 
-  "&:hover": {
-    backgroundColor: "#fff",
-  },
+	"&:hover": {
+		backgroundColor: "#fff",
```
Comment (dianamiraute) [inline src/components/common/card/CardStyle.js:10]: You still have some wrong indentation

File: src/pages/homepage/HomePage.js [hunk 3]
```diff
@@ -2,57 +2,60 @@ import Header from "../../components/common/header/Header";
...
+	};
+
+	const sortedDate = [...CardsData].sort(
+		(objA, objB) => Number(new Date(objB.date)) - Number(new Date(objA.date))
+	);
+
+	return (
+		<div>
+			<Container maxWidth="lg">
+				<Header />
```
Comment (dianamiraute) [inline src/pages/homepage/HomePage.js:32]: Fix indentation

Comment (TirlaP) [reply src/pages/homepage/HomePage.js:32]: I tried to fix the identation from GitHub, but it's still not working. In the editor it's perfect

File: src/components/common/card/CardStyle.js (hunk 2 above)
Comment (TirlaP) [reply src/components/common/card/CardStyle.js:10]: I tried to fix the identation from GitHub, but it's still not working. In the editor it's perfect

**Pull request merged**